### ISSUE TITLE
Air-gap - add alpine image to rancher-images.txt

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#21650]
Bug
 
The image `alpine` is needed for upgrading the single-node-installed rancher servers where backup needs to be taken in the air-gapped environment. 

It will be good if we can include the image in the `rancher-images.txt` file shipped with Rancher's release.

**Comment by alena1108** [comment]
@jiaqiluo what alpine version do you use? Is it documented anywhere?

**Comment by leodotcloud** [comment]
Since we use busybox image as part of the helm chart for installing rancher, may be we can use that image for backup as well instead of alpine. That way we can add just busybox image to the rancher-images.txt and skip alpine.

**Comment by leodotcloud** [comment]
@alena1108 alpine image is being used in the docs here: https://rancher.com/docs/rancher/v2.x/en/backups/backups/single-node-backups/

**Comment by leodotcloud** [comment]
Tested with the following steps:
```
docker run -d --name ranchertest --restart=unless-stopped -p 8080:80 -p 8443:443 rancher/rancher:v2.2.6 --no-cacerts
docker create --volumes-from ranchertest --name rancher-data-jul29 rancher/rancher:v2.2.6
docker run  --volumes-from rancher-data-jul29 -v $PWD:/backup busybox tar zcvf /backup/rancher-data-backup-v221-jul29.tar.gz /var/lib/rancher
```

There is no specific dependency on alpine image, busybox works just fine.

**Comment by jiaqiluo** [comment]
The bug fix is validated in rancher:v2.3.0-alpha7

Confirm that the image `busybox` is in the `rancher-images.txt` file (https://github.com/rancher/rancher/releases/tag/v2.3.0-alpha7)

```
> cat rancher-images.txt | grep busybox
busybox
```
